src/Shared/Controls/Settings/Ipfs/IpfsSettings.xaml

```diff
@@ -290,7 +290,7 @@
 							<ProgressRing IsActive="{x:Bind PinCommand.IsRunning, Mode=OneWay}" />
 							<ProgressRing IsActive="{x:Bind UnpinCommand.IsRunning, Mode=OneWay}" />
 
-							<ToggleSwitch x:Name="PreloadToggleSwitch" IsOn="{x:Bind IsPinned, Mode=TwoWay}" />
+							<ToggleSwitch x:Name="PreloadToggleSwitch" IsOn="{Binding IsPinned, Mode=TwoWay}" />
```

Comment:
Why not x:Bind, is the DataTemplate not fetching the datatype or because `Binding` is more reliable with the TwoWay mode?

Reply:
Encountered an error with Uno on this one property 🤷